tests/test_DataAugmenterExternally.py

```diff
@@ -0,0 +1,66 @@
+import os
+import pytest
+import pandas as pd
+
+import sys
+
+sys.path.append("zarnitsa/")
+
+from DataAugmenterInternally import DataAugmenterInternally
+
+
+@pytest.fixture
+def dataug_internally():
+    return DataAugmenterInternally()
+
+
+def test_augment_column_permute(dataug_internally):
```

Comment:
You shouldn't include test data into test function body, use fixtures to send data into test function.
Also one execution should check one case, not several at the same time.
Also you shouldn't check raising exception via `assert False`, use `with pytest.raises(ValueError)` instead.